README.md

```diff
@@ -5,5 +5,6 @@ This is a ruby codebase which has some tests, but not perfect coverage.
 Run the following to run the tests and report coverage:
 
 ```shell
-CODECLIMATE_REPO_TOKEN=TOKENGOESHERE bundle exec rspec
+export CODECLIMATE_REPO_TOKEN=TOKENGOESHERE
+bundle exec rspec && bundle exec codeclimate-test-reporter
```

Comment:
FWIW, this would also work:

```console
bundle exec rspec && \
  CODECLIMATE_REPO_TOKEN=TOKENGOESHERE bundle exec codeclimate-test-reporter
```

And doesn't pollute the local environment.